client/asset/dcr/dcr.go

```diff
@@ -548,6 +568,12 @@ type findRedemptionResult struct {
 // NewWallet is the exported constructor by which the DEX will import the
 // exchange wallet.
 func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network) (*ExchangeWallet, error) {
+	// store the network because we might need it for external api calls.
+	if network == dex.Testnet {
+		isTestnet = true
+	} else {
+		isTestnet = false
+	}
```

Comment:
No globals here, please. Let's pass it in to `unconnectedWallet` and save it as a field on the `ExchangeWallet`.